UNIT_TESTS.md

```diff
@@ -0,0 +1,74 @@
+# Unit Tests
+
+Besides the code of the module itself, it is important to submit in addition a
+suite of tests. They guarantee your add-on remains functional when changes are
+applied to the code on which it depends. Once committed to GitHub, the unit
+tests are run automatically by the CI/CD pipeline every time the code in the
+repository is modified. Notifications are issued if a modification somehow
+impacts your add-on.
+
+## Requirements
+
+The unit tests depend on software that you must install in your system
+beforehand: Subversion and the GRASS development files. On a packaged operating
+system like Ubuntu or Debian Linux this can be achieved with:
+
+```bash
+sudo apt install subversion grass-dev
+```
+
+If you have not done so yet, it is also helpful to install pre-commit. Follow
+[the
+instructions](https://github.com/OSGeo/grass/blob/main/doc/development/submitting/submitting.md#use-pre-commit)
+in the main GRASS repository.
+
+## Create a unit test
+
+Unit tests must be stored in a subfolder named `testsuite` inside the add-on
+folder. For example, if you contributed a raster module named r.mymodule you
+would run:
+
+```bash
+cd src/raster/r.mymodule
+mkdir testsuite
+```
+
+Then you can create a new Git branch and initiate a new unit test class:
+
+```bash
+git checkout new-branch
+vim testsuite/test_mymodule.py
+```
+
+The [unit testing
+documentation](https://grass.osgeo.org/grass82/manuals/libpython/gunittest_testing.html)
```

Comment:
can we change grass82 to grass-stable so it's always up to date?

Reply:
Fixed links in 00b011192429dd0c6a494d9364ef642e465c832c